src/screens/SignIn/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+
+import nimbleLogoWhite from 'assets/images/icons/nimble-logo-white.svg';
+import ElevatedButton from 'components/ElevatedButton';
+import TextInput from 'components/TextInput';
+
+export const signInScreenTestIds = {
+  nimbleLogo: 'sign-in__nimble-logo',
+  signInForm: 'sign-in-form',
+  emailLabel: 'sign-in-form__email',
+  emailField: 'sign-in-form__input-email',
+  passwordLabel: 'sign-in-form__password',
+  passwordField: 'sign-in-form__input-password',
+  forgotButton: 'sign-in-form__forgot-button',
+  signInButton: 'sign-in-form__button',
+};
+
+const SignInScreen = (): JSX.Element => {
+  return (
+    <div className="sign-in bg-cover min-h-screen flex flex-col justify-center items-center bg-sign-in">
+      <img className="mx-auto" src={nimbleLogoWhite} alt="nimble logo" data-test-id={signInScreenTestIds.nimbleLogo} />
+      <div className="mt-6 mb-8 text-white opacity-60 text-regular tracking-survey-tight">Sign in to Nimble</div>
+      <form className="w-80" data-test-id={signInScreenTestIds.signInForm}>
+        <div className="mb-6">
+          <TextInput
+            label="Email"
+            labelDataTestId={signInScreenTestIds.emailLabel}
+            inputAttributes={{
+              id: 'form-sign-in-email',
+              required: true,
+              type: 'email',
+              'data-test-id': signInScreenTestIds.emailField,
+            }}
+          />
+        </div>
+        <div className="mb-6 relative">
+          <span className="block w-full">
+            <TextInput
+              label="Password"
+              labelDataTestId={signInScreenTestIds.passwordLabel}
+              inputAttributes={{
+                id: 'form-sign-in-password',
+                required: true,
+                type: 'password',
+                'data-test-id': signInScreenTestIds.passwordField,
+              }}
+              extraClassName="pr-16"
```

Comment:
we could just name it as `className`,  it's a common naming convention, for example in React codebase:

https://github.com/facebook/react/blob/a1c62b8a7635c0bc51e477ba5437df9be5a9e64f/packages/react-devtools-shared/src/devtools/views/Icon.js#L32

Reply:
got it @tyrro 🚀

Reply:
@tyrro resolved in [a31c486](https://github.com/manh-t/react-survey/pull/42/commits/a31c486e6aabdb861be574b7d76fd7ec5686a78d)